packages/arb-token-bridge-ui/src/util/networks.ts

```diff
@@ -499,9 +524,8 @@ export function getDestinationChainIds(chainId: ChainId): ChainId[] {
   const validDestinationChainIds = getChildChainIds(chain)
 
   if (parentChainId) {
-    // always make parent chain the first element
-    return [parentChainId, ...validDestinationChainIds]
+    return sortChainIds([parentChainId, ...validDestinationChainIds])
   }
 
-  return validDestinationChainIds
+  return sortChainIds(validDestinationChainIds)
```

Comment:
looks like `sortChainIds` can be used directly in `getChildChainIds(chains)`?

I don't think we need to sort the `parentChainId` away from its first position